pkg/operatorrules/registry.go

```diff
@@ -65,7 +67,10 @@ func ListAlerts() []promv1.Rule {
 	}
 
 	slices.SortFunc(alerts, func(a, b promv1.Rule) int {
-		return cmp.Compare(a.Alert, b.Alert)
+		aKey := a.Alert + ":" + a.Expr.String()
+		bKey := b.Alert + ":" + b.Expr.String()
+
+		return strings.Compare(aKey, bKey)
```

Comment:
why this change? and this is for alerts

Reply:
Changed to ListRecordingRules().
And it's since the tests checks the ListRecordingRules length, and if we didn't change it here it will not list the rule twice in case of same name and different expr:  https://github.com/machadovilaca/operator-observability/pull/9/files/ccb0c0cb0001fd248a9704445ae7fe07ecc66be8#diff-6002554c8c543720622e7cc8dd82f6d70e0876e73e61eb667796658430760bcdR80